### ISSUE TITLE
Is this exponential backoff correct?

### ISSUE THREAD

**Issue by ivashov** [yigit/android-priority-jobqueue#33]
```Java
(long) Math.pow(initialBackOffInMs, runCount)
```

For example with initialBackOffInMs=1000ms it result in next delays:
```
1000
1000000
1000000000
```

i think backoff function should have at least factor parameter, for example
```
    public static RetryConstraint createExponentialBackoff(int runCount, long initialBackOffInMs, double factor) {
        RetryConstraint constraint = new RetryConstraint(true);
        constraint.setNewDelayInMs((long) (initialBackOffInMs * Math.pow(factor, runCount)));
        return constraint;
    }
```

Some code: http://ideone.com/mlb1t9

May be i wrong, though

**Comment by mseshachalam** [comment]
+1 good observation

**Comment by yigit** [comment]
yes, stupid me :/, it should just double. I'll change it, meanwhile, I assume it is not a blocker since you can already provide your own.

**Comment by yigit** [comment]
released 1.3.3 with this fix. Thanks.
